modules/op-release-notes-1-1.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assembly:
+//
+// * openshift_pipelines/op-release-notes.adoc
+
+[id="op-release-notes-1-1_{context}"]
+= Release notes for Red Hat {pipelines-title} Technology Preview 1.1
+
+[id="new-features-1-1_{context}"]
+== New features
+{pipelines-title} Technology Preview (TP) 1.1 is now available on {product-title} {product-version}. {pipelines-title} TP 1.1 is updated to support:
+
+* Tekton Pipelines 0.14.3
+* Tekton `tkn` CLI 0.11.0
+* Tekton Triggers 0.6.1
+* ClusterTasks based on Tekton Catalog 0.14
+
+In addition to the fixes and stability improvements, here is a highlight of what’s new in OpenShift Pipelines 1.1.
+
+=== Pipelines
+* Workspace support for VolumeClaimTemplates has been added:
+** The VolumeClaimTemplate for a PipelineRun and TaskRun can now be added as a volume source for Workspaces. The tekton-controller then creates a PersistentVolumeClaim (PVC) using the template that is seen as a PVC for all TaskRuns in the Pipeline. Thus you do not need to define the PVC configuration every time it binds a workspace that spans multiple tasks.
+** Support to find the name of the PersistentVolumeClaim when a VolumeClaimTemplate is used as a volume source is now added using variable substitution.
+* Support for improving audits:
+** The `PipelineRun.Status` field now contains the status of every TaskRun in the Pipeline and the Pipeline specification used to instantiate a PipelineRun to monitor the progress of the PipelineRun.
+** Pipeline results are now added to the pipeline specification and `PipelineRun` status.
+** The `TaskRun.Status` field, now contains the exact Task specification used to instantiate the `TaskRun`.
+* Support to apply the default parameter to Conditions.
+* A TaskRun created by referencing a ClusterTask now adds the `tekton.dev/clusterTask` label instead of the `tekton.dev/task` label.
+* The `kubeconfigwriter` now adds the `ClientKeyData` and the `ClientCertificateData` configurations in the Resource structure to enable replacement of the pipeline resource type cluster with the kubeconfig-creator Task.
+* The feature-flags config map names and the defaults config map name are now customizable.
+* Support for HostNetwork in the PodTemplate used by TaskRun is now added.
+* An Affinity Assistant is now introduced to support node affinity in TaskRuns that share workspace volume. By default, this is disabled on OpenShift Pipelines.
```

Comment:
Same comment as above, just with "is now introduced". Suggest "is now available" or "has been introduced"